server/server.go

```diff
@@ -69,42 +67,13 @@ const (
 
 // ArgoRolloutsServer holds information about rollouts server
 type ArgoRolloutsServer struct {
-	Options     ServerOptions
-	NamespaceVC NamespaceViewController
```

Comment:
@huikang this line `NamespaceVC NamespaceViewController` is
already removed from the `ArgoRolloutsServer`.

 It's related to another comment. Just maybe  you're looking at a different version of the file